packages/components/ovh-payment-method/src/constants.js

```diff
@@ -0,0 +1,52 @@
+import { PAYMENT_MEAN_TYPE_ENUM } from './enums/payment-mean.enum';
+
+export const AVAILABLE_PAYMENT_MEAN_TYPES = {
```

Comment:
no entry for US?

Reply:
Nope, there is no notions of payment mean in US ;-)